### DISCUSSION TITLE
After Update zero-template 3.0.12 - Error site loading

### DISCUSSION THREAD

**Discussion by knut32** [templaza/astroid-framework#659] [Q&A]
Since the update to version 3.0.12 my page no longer loads.
I get the message:

Oops!!! Page not found.

It seems like the page that you are looking for is no longer here.
Home page

When I click on the Home button, I get the same error.
But if I switch off the menu, the page loads.

**Comment by Chacapamac** [comment]
Did You try to empty all caches (Browser - Joomla - Astroid)
In case like that, I even go directly by FTP and delete all caches files manually.
